ios/FluentUI/Navigation/BadgeLabelButton.swift

```diff
@@ -208,5 +208,19 @@ class BadgeLabelButton: UIButton {
 
     @objc private func badgeValueDidChange() {
         updateBadgeLabel()
+        updateAccessibilityLabel()
+    }
+
+    private func updateAccessibilityLabel() {
+        guard let item = item else {
+            return
+        }
+        if let itemAccessibilityLabel = item.accessibilityLabel, let badgeAccessibilityLabel = item.badgeAccessibilityLabel {
+            accessibilityLabel = String(format: "Accessibility.BadgeLabelButton.LabelFormat".localized,
```

Comment:
Can you use [localizedStringWithFormat(_:_:)](https://developer.apple.com/documentation/foundation/nsstring/1408481-localizedstringwithformat) instead of [init(format:arguments:)](https://developer.apple.com/documentation/foundation/nsstring/1407827-init)?

We made the standard to use the former because [init(format:arguments:)](https://developer.apple.com/documentation/foundation/nsstring/1407827-init) does not do any localization while formatting.

Reply:
Ohh!! I didn't know about this!!